test/local/log_tests.js

```diff
@@ -419,28 +420,57 @@ test(
 
 test(
   'log.flowEvent with a bad request',
-  function (t) {
-    return log.flowEvent('account.login').then(function () {
+  t => {
+    return log.flowEvent('account.signed').then(() => {
       t.equal(logger.error.callCount, 1, 'logger.error was called once')
-      var args = logger.error.args[0]
+      const args = logger.error.args[0]
       t.equal(args[0], 'log.flowEvent', 'correct op')
-      t.equal(args[1].event, 'account.login', 'correct event name')
+      t.equal(args[1].event, 'account.signed', 'correct event name')
       t.equal(args[1].badRequest, true, 'correct flag')
 
       t.equal(logger.debug.callCount, 0, 'logger.debug was not called')
       t.equal(logger.critical.callCount, 0, 'logger.critical was not called')
       t.equal(logger.warn.callCount, 0, 'logger.warn was not called')
       t.equal(logger.info.callCount, 0, 'logger.info was not called')
+      t.equal(metricsContext.gather.callCount, 0, 'metricsContext.gather was not called')
 
       logger.error.reset()
     })
   }
 )
 
 test(
+  'log.flowEvent with content server account.signed event',
+  t => {
+    return log.flowEvent('account.signed', {
+      headers: {
+        'user-agent': 'foo'
+      },
+      payload: {
+        metricsContext: {
+          flow_id: 'bar',
```

Comment:
It does, however the use of  `flow_id` here is an artifact of the naive behaviour of the test stub for `metricsContext.gather`, defined above on line 22. It just returns the payload unadulterated, whereas the real implementation maps the property names to the ones we use on the event, including `flowId` to `flow_id`.

So use of the latter here was to conform with behaviour of the test stub rather than the Joi schema, which these tests don't go through. But I can see that is totally confusing to a casual eyeballer of the test code, so I'll improve the stub implementation and update the payloads.